lib/msal-core/samples/VanillaJSTestApp/index_LoginPopup.html

```diff
@@ -54,14 +64,23 @@ <h1>Sending an email with msal.js and Microsoft Graph</h1>
         }
 
         function loginPopup() {
-            userAgentApplication.loginPopup(applicationConfig.graphScopes).then(function (idToken) {
+            let tokenRequest = {
+              scopes: applicationConfig.graphScopes
+              // loginHint: "idlab@msidlab4.microsoft.com",
```

Comment:
if we are leaving in samples, it might be good to have a comment like this

// uncomment to test x flow

Reply:
Samples can be taken up in a different PR.